perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -0,0 +1,473 @@
+
+
+def UnmountGCSBucket(gcs_bucket) -> None:
+  """Unmounts the GCS bucket.
+
+  Args:
+    gcs_bucket: Name of the directory to which GCS bucket is mounted to.
+
+  Raises:
+    An error if the GCS bucket cound not be umnounted.
+  """
+  log.info('Unmounting the GCS Bucket.\n')
+
+  exit_code = subprocess.call('umount -l {}'.format(gcs_bucket), shell=True)
+
+  if exit_code != 0:
+    log.error('Error encountered in umounting the bucket. Aborting!\n')
+    subprocess.call('bash', shell=True)
+
+  else:
+    subprocess.call('rm -rf {}'.format(gcs_bucket), shell=True)
+    log.info('Successfully unmounted the bucket and deleted %s directory.\n', gcs_bucket)
+
+
+def MountGCSBucket(bucket_name) -> str:
+  """Mounts the GCS bucket into the gcs_bucket directory.
+
+  Args:
+    bucket_name: Name of the bucket to be mounted.
+
+  Returns:
+    A string which contains the name of the directory to which the bucket is mounted.
+
+  Raises:
+    An error into the logs if cannot mount the bucket due to some error. Also shut offs the program.
+  """
+  log.info('Started mounting the GCS Bucket using GCSFuse.\n')
+  gcs_bucket = bucket_name
+
+  subprocess.call('mkdir {}'.format(gcs_bucket), shell=True)
+
+  exit_code = subprocess.call('gcsfuse --implicit-dirs --disable-http2 --max-conns-per-host 100 {} {}'.format(
+      bucket_name, gcs_bucket), shell=True)
+  print('')
+
+  if exit_code != 0:
+    log.error('Cannot mount the GCS bucket due to exit code %s.\n', exit_code)
+    subprocess.call('bash', shell=True)
+  else:
+    return gcs_bucket
+
+
+def ParseArguments(argv):
+  """Parses the arguments provided to the script via command line.
+
+  Args:
+    argv: List of arguments recevied by the script.
+
+  Returns:
+    A class containing the parsed arguments.
+  """
+  argv = sys.argv
+
+  parser = argparse.ArgumentParser()
+
+  parser.add_argument(
+      'config_file',
+      help='Provide path of the config file.',
+      action='store'
+  )
+
+  parser.add_argument(
+      '--keep_files',
+      help='Does not delete the directory structure in persistent disk.',
+      action='store_true',
+      default=False,
+      required=False,
+  )
+
+  parser.add_argument(
+      '--upload',
+      help='Upload the results to the Google Sheet.',
+      action='store_true',
+      default=False,
+      required=False,
+  )
+
+  parser.add_argument(
+      '--message',
+      help='Puts a message/title describing the test.',
+      action='store',
+      nargs=1,
+      default='Performance Listing Benchmark',
+      required=False,
+  )
+
+  parser.add_argument(
+      '--num_samples',
+      help='Number of samples to collect of each test.',
+      action='store',
+      nargs=1,
+      default=10,
+      required=False,
+  )
+
+  parser.add_argument(
+      '--command',
+      help='Command to run the tests on.',
+      action='store',
+      nargs=1,
+      default='ls -R',
+      required=True,
+  )
+
+  args = parser.parse_args(argv[1:])
+
```

Comment:
nit: remove new line or you can directly call return parser.parse_args(argv[1:])

also can you add a comment why you are ignnoring first parameter.